shenyu-plugin/shenyu-plugin-redirect/src/main/java/org/apache/shenyu/plugin/redirect/RedirectPlugin.java

```diff
@@ -90,11 +90,13 @@ protected Mono<Void> doExecute(final ServerWebExchange exchange, final ShenyuPlu
 
     @Override
     protected Mono<Void> handleSelectorIfNull(final String pluginName, final ServerWebExchange exchange, final ShenyuPluginChain chain) {
-        return WebFluxResultUtils.noSelectorResult(pluginName, exchange);
+        LOG.error("can not match selector data: {}", pluginName);
```

Comment:
not add this

Reply:
Do you need to print logs that do not match the select?

Reply:
not modify this，not log

Reply:
I have removed it.